api/dbadapters/bigquery.ts

```diff
@@ -190,6 +195,20 @@ export class BigQueryDbAdapter implements IDbAdapter {
 
   public async close() {}
 
+  public async persistedStateMetadata(
+    database: string
```

Comment:
we shouldn't need this. queries always run by default in the scope of the database to which we're connected, which is the default database.